internal/api/api_client.go

```diff
@@ -48,21 +47,19 @@ func NewAPIClient(nozzleConfig *config.NozzleConfig) (*APIClient, error) {
 		apiURL = "https://" + apiURL
 	}
 
-	apiConfig := &cfclient.Config{
+	// TODO: rename Username to ClientID
```

Comment:
Do we want to do this now or leave for later? Guess this would need a change to the tile as well.